packages/manager/apps/dedicated/client/app/account/user/users/groups/update/user-users-groups-update.controller.js

```diff
@@ -0,0 +1,54 @@
+import get from 'lodash/get';
+
+export default class UserAccountUsersGroupsUpdateCtrl {
+  /* @ngInject */
+  constructor(
+    $scope,
+    coreConfig,
+    UseraccountGroupsService,
+    Alerter,
+    $translate,
+  ) {
+    this.$scope = $scope;
+    this.groupsService = UseraccountGroupsService;
+    this.alerter = Alerter;
+    this.$translate = $translate;
+    this.me = coreConfig.getUser();
```

Comment:
**question:** Still needed? (can't find where it's used)

Reply:
this.alerter, this.$translate and this.me removed